components/Navbar.vue

```diff
@@ -31,7 +31,7 @@
           Team
         </nuxt-link>
         <nuxt-link
-          to="/"
+          to="https://wiki.cybermods.net"
```

Comment:
If link points to external site it should be using `<a>` tag instead of `<nuxt-link>`. Now it will still point to `/` as `<nuxt-link>` works only internally

Reply:
didnt know, good to know for future